AirCasting/SessionViews/SessionCartViewModel.swift

```diff
@@ -0,0 +1,47 @@
+// Created by Lunar on 29/07/2021.
+//
+
+import Foundation
+
+class SessionCartViewModel: ObservableObject {
+    private let sessionCartFollowing: SessionCartFollowing
+    @Published var isFollowing: Bool = false
```

Comment:
I don't think we should have a default here, we need to determine correct value in `init`